Index: meson.build
<+>UTF-8
===================================================================
diff --git a/meson.build b/meson.build
--- a/meson.build	(revision df4240ec1dff31fee8ad053ad7693b12d6e7064a)
+++ b/meson.build	(date 1610834054000)
@@ -10,7 +10,9 @@

 icecc = dependency('icecc')
 glib = dependency('glib-2.0')
+
 ncurses = dependency('ncursesw')
+ncurses_cxxflag = '-DNCURSES_WIDECHAR'

 cxx = meson.get_compiler('cpp')

@@ -33,8 +35,9 @@
 if not cxx.links(code, dependencies: deps, name: 'libicecc links dynamically')
     icecc = dependency('icecc', static: true)
     liblzo2 = cxx.find_library('lzo2')
-    libcapng = dependency('libcap-ng')
-    deps += [libcapng, liblzo2]
+    libcapng = dependency('libcap-ng', required: false)
+    libzstd = cxx.find_library('zstd', required: false)
+    deps += [liblzo2, libcapng, libzstd]
 endif

 incdir = include_directories('src')
@@ -64,7 +67,8 @@
     ['src/main.cpp', 'src/draw.cpp', 'src/scheduler.cpp', 'src/simulator.cpp'],
     include_directories: incdir,
     dependencies: deps,
-    install : true
+    install : true,
+    cpp_args: ncurses_cxxflag
     )

 # A pretty dumb test right now. Simply run the simulator with a known key for a
