diff --git a/meson.build b/meson.build
index 3354f10..f401756 100644
--- a/meson.build
+++ b/meson.build
@@ -24,8 +24,7 @@ dep_systemd = dependency('libsystemd', version: '>218', required: false)
 dep_tinyxml2 = dependency('tinyxml2')
 dep_jsoncpp = dependency('jsoncpp')
 
-libfmt = subproject('fmt', default_options: ['header-only=true'])
-dep_libfmt = libfmt.get_variable('fmt_header_only_dep')
+dep_libfmt = dependency('fmt')
 
 # This is not a link dependency for this project, but we need it
 # to get the proper paths for policy and service config installation
